web/models/build_status_updater.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Bigben.BuildStatusUpdater do
+  alias Bigben.Build
+  alias Bigben.Repo
+
+  def update(build_params) do
+    changeset = Build.changeset(%Build{}, build_params)
+    Repo.insert!(changeset)
+  end
```

Comment:
This is likely premature optimization but the next thing that is going to happen is we need to make a call back to the Travis API to get the individual run times for each job inside the build. I think pulling that out of the controller makes sense.

Otherwise, I completely agree with you.